projects/igniteui-angular/src/lib/grids/resizing/resize-handle.directive.ts

```diff
@@ -21,86 +23,74 @@ import { IgxColumnResizingService } from './resizing.service';
     selector: '[igxResizeHandle]',
     standalone: true
 })
-export class IgxResizeHandleDirective implements AfterViewInit, OnDestroy {
+export class IgxResizeHandleDirective implements AfterViewInit {
+    private _ref: ElementRef<HTMLElement> = inject(ElementRef);
+    private destroyRef = inject(DestroyRef);
+    protected zone = inject(NgZone);
+    protected colResizingService = inject(IgxColumnResizingService);
+
+    private readonly DEBOUNCE_TIME = 200;
+
+    protected get element() {
+        return this._ref.nativeElement;
+    }
 
     /**
      * @hidden
      */
     @Input('igxResizeHandle')
     public column: ColumnType;
 
-    /**
-     * @hidden
-     */
-    protected _dblClick = false;
 
-    /**
-     * @hidden
-     */
-    private destroy$ = new Subject<boolean>();
 
-    private readonly DEBOUNCE_TIME = 200;
-
-    constructor(protected zone: NgZone,
-        protected element: ElementRef,
-        public colResizingService: IgxColumnResizingService) { }
 
     /**
      * @hidden
      */
     @HostListener('dblclick')
     public onDoubleClick() {
-        this._dblClick = true;
         this.initResizeService();
         this.colResizingService.autosizeColumnOnDblClick();
     }
 
-    /**
-     * @hidden
-     */
-    public ngOnDestroy() {
-        this.destroy$.next(true);
-        this.destroy$.complete();
-    }
+
 
     /**
      * @hidden
      */
     public ngAfterViewInit() {
         if (!this.column.columnGroup && this.column.resizable) {
             this.zone.runOutsideAngular(() => {
-                fromEvent(this.element.nativeElement, 'mousedown').pipe(
-                    debounceTime(this.DEBOUNCE_TIME),
```

Comment:
The removal of the debounceTime and check for dblClick kinda changes the behavior for auto size when double clicking a column. Comparison old vs new: